src/components/BaseToast.tsx

```diff
@@ -36,7 +36,7 @@ export function BaseToast({
         testID={getTestId('ContentContainer')}
         style={[styles.contentContainer, contentContainerStyle]}
         {...contentContainerProps}>
-        {text1 && text1.length > 0 && (
+        {text1 && text1.length > 0 ? (
```

Comment:
Let's do it like this
```suggestion
        {text1?.length > 0 ? (
```